Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN apt install -y git gcc g++ make cmake autoconf automake libtool python3 libs
 WORKDIR /kvrocks
 
 COPY . .
-RUN ./x.py build -DENABLE_OPENSSL=ON
+RUN ./x.py build -DENABLE_OPENSSL=ON -DPORTABLE=ON
```

Comment:
I'd suggest adding an `ARG` and defaulting to `ON`.

Our image can always use the default but if users are certain for the platform they can build their own with `PORTABLE=OFF`.

Reply:
done. since other build arguments can also have this problem like ENABLE_SSL, DISABLE_JEMALLOC, so I add $MORE_BUILD_ARGS instead of $PORTABLE for more general purpose.